services/worker-manager/src/providers/google.js

```diff
@@ -288,17 +278,21 @@ class GoogleProvider extends Provider {
                 'https://www.googleapis.com/auth/cloud-platform',
               ],
             }],
+            scheduling: {
+              ...cfg.scheduling || {},
+              automaticRestart: false,
```

Comment:
It's probably worth mentioning this and the other hard-coded things (shutdown?) in the schema.

Reply:
We can't hardcode shutdown unfortunately because the different workers get configured to do that differently. Perhaps we should investigate making worker-runner handle that part in a uniform way?

I'll add notes about the other hard-coded things in the schema

I tested this in my dev cluster with old non-compliant worker pools. It throws an error but otherwise doesn't bust anything. I figure if we land this, then the ci-config changes within a few minutes it will be fine.